src/pretix/control/forms/checkin.py

```diff
@@ -177,3 +178,26 @@ class Meta:
             'subevent': SafeModelChoiceField,
             'gates': SafeModelMultipleChoiceField,
         }
+
+
+class CheckinListSimulatorForm(forms.Form):
+    raw_barcode = forms.CharField(
+        label=_("Barcode"),
```

Comment:
Hmm, at the other places in the pretix backend, we call this `ticket code`.

Reply:
True... But I would assume that people who use the simulator do indeed scan a barcode to check the system...

I see this one valid one or the other way :-)